dashboard/app/models/studio_ec.rb

```diff
@@ -21,7 +21,7 @@
 #
 
 class StudioEC < Studio
-  before_save :update_palette
+  before_save :update_palette, :update_goalOverride
```

Comment:
@cpirich Do you know how I can add support for parsing a goal_override object inside the .level file?